source/manual/alerts/nginx-requests-too-low.html.md

```diff
@@ -4,16 +4,16 @@ title: Nginx requests too low
 parent: "/manual.html"
 layout: manual_layout
 section: Icinga alerts
-last_reviewed_on: 2019-03-05
+last_reviewed_on: 2019-09-05
 review_in: 6 months
 ---
 
 We monitor the number of requests reaching our Nginx servers. We expect that
 there will be a minimum number of requests occurring and a check will alert if
 this falls below a threshold.
 
-[View the Nginx request logs](nginx_grafana) or view the impacted
-application dashboard on Grafana.
+View the Nginx logs in Kibana or view the impacted application dashboard on
```

Comment:
Is it not more useful to have a link?

Reply:
Agreed, at the time I didn't find a sensible link. I have now, and will be updating shortly.